headers/functions/overlay29.h

```diff
@@ -181,6 +204,13 @@ void RestoreMovePP(struct entity* user, struct entity* target, int pp, bool supp
 void SetReflectDamageCountdownTo4(struct entity* entity);
 bool HasConditionalGroundImmunity(struct entity* entity);
 int Conversion2IsActive(struct entity* entity);
+int AiConsiderMove(struct ai_possible_move* ai_possible_move, struct entity* monster,
+                   struct move* move);
+int TryAddTargetToAiTargetList(int current_num_targets, struct move_target_and_range,
```

Comment:
Second param is missing a name

Reply:
Fixed! I didn't even know that was valid syntax

Reply:
Yeah in C function declarations don't need parameter names. But some styles call for it for neatness, and it also helps Ghidra to include names.